docs/howto/getting-started/accessing-cc-rest-api.md

```diff
@@ -0,0 +1,189 @@
+# Accessing the Cleura Cloud REST API
+
+{{brand}} provides a REST API, which you can take advantage of with a
+tool like `curl`. But before you do, you must create a token for the
+current session. For that, you only need to have an
+[account in {{brand}}](/howto/getting-started/create-account).
+
+## Creating a token
+
+If you do not have two-factor authentication (2FA) enabled for your
+account, to create a token, have your username (`your_cc_username`)
+and password (`your_cc_password`) to the {{brand}} handy and type:
+
+```bash
+curl -H "Content-Type: application/json" \
+    -d '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
+    https://rest.cleura.cloud/auth/v1/tokens
+```
+
+Provided you typed your username and password correctly and that there
+were no connection issues to the remote endpoint, you will get a JSON
+object with a string (`token`) that holds your session token:
+
+```json
+{
+  "result": "login_ok",
+  "token": "vahkie7EiDaij7chegaitee2zohsh1oh"
+}
+```
+
+If, on the other hand, you do have 2FA enabled for your account,
+then first request a token as you would without 2FA enabled:
+
+```bash
+curl -H "Content-Type: application/json" \
+    -d '{"auth": {"login": "your_cc_username", "password": "your_cc_password"}}' \
+    https://rest.cleura.cloud/auth/v1/tokens
+```
+
+Instead of a token, you will get a verification code
+(look at `verification`):
+
+```json
+{
+  "result": "twofactor_required",
+  "type": "sms",
+  "verification": "ahb4en3cho"
+}
+```
+
+This verification code is required for requesting a 2FA code:
+
+```bash
+curl -X POST \
+    -d '{"request2fa": {"login": "your_cc_username", "verification": "ahb4en3cho"}}' \
+    https://rest.citycloud.com/auth/v1/tokens/request2facode
+```
+
+In our example, the code is sent via an SMS message (check the JSON
+object above and, more specifically, look at the value of `type`).
+So after getting a new SMS message with your 2FA code, which expires
+after ten minutes, go ahead and request your token like this:
+
+```bash
+curl -X POST \
+    -d '{"verify2fa": {"login": "your_cc_username", "verification": "ahb4en3cho", "code": 2fa_code}}' \
+    https://rest.citycloud.com/auth/v1/tokens/verify2fa
+```
+
+(Make sure not to put the `2fa_code` in quotes, for it is
```

Comment:
You can drop the parentheses around this sentence.

Reply:
Those are gone, also.